tests/everest/conftest.py

```diff
@@ -1,568 +1,84 @@
-import fileinput
```

Comment:
From the diff, a lot of stuff was removed from this file. Where did it go? Is it now mostly imported from ERT? All the qt related fixtures for example. Not  obvious from the diff, guessing it is now also getting that from ERT?

Reply:
It is mostly a git thing, as the `conftest.py` from ert and everest are very similar, so it thinks this was moved and then deleted (confusing it with the `ert` conftest.py. Will see if I can improve the diff.